collectd-docker/README.md

```diff
@@ -66,6 +70,17 @@ The [`docker-collectd`](https://github.com/signalfx/docker-collectd-plugin) plug
 
 1. Restart collectd.
 
+1. Optional metrics regarding CPU quota and CPU shares can be enabled in the plugin configuration file. To enable the optional metrics:
+  * Set the CpuQuotaPercent and CpuSharesPercent configuration options to true
+  * Configure the filter to emit the optional metrics. Please see the configuration details [here](#configuration)
+
+5. The optional dashboard `Docker Neighbours` offers visualizations based on the CPU quota and CPU shares metrics. To view the dashboard:
+  * Manually import the dashboard into your organization in SignalFx.
+    * Download the dashboard
+    * In SignalFx, select `hamburger menu` -> `import` -> `Dashboard Group`
```

Comment:
I cleared it with barbara-sfx.  Change this and the  to "Click to open the Actions menu, hover over Import, then select DashboardGroup."